site/content/docs/5.0/components/dropdowns.md

```diff
@@ -896,6 +896,16 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td><code>'dynamic'</code></td>
       <td>By default, we use Popper for dynamic positioning. Disable this with <code>static</code>.</td>
     </tr>
+    <tr>
+      <td><code>offset</code></td>
+      <td>number | string | function</td>
+      <td><code>"0,0"</code></td>
```

Comment:
Default value for offset is array @joke2k :slightly_smiling_face: 
```js
offset: [0, 0],
```

Reply:
I changed a little bit the copy